module/metrics/execution.go

```diff
@@ -739,6 +749,11 @@ func (ec *ExecutionCollector) ExecutionTransactionExecuted(
 	ec.transactionExecutionTime.Observe(float64(dur.Milliseconds()))
 	ec.transactionConflictRetries.Observe(float64(numConflictRetries))
 	ec.transactionComputationUsed.Observe(float64(compUsed))
+	if compUsed > 0 {
+		// normalize so the value should be around 1
+		ec.transactionNormalizedTimePerComputation.Observe(
+			(float64(dur.Milliseconds()) / float64(compUsed)) * flow.EstimatedComputationPerMillisecond)
```

Comment:
I still don't quite get this math.  According to this comment "Value below 1 means the transaction was executed faster than estimated (is using less resources then estimated)"

So if EstimatedComputationPerMillisecond is `50`, it means normally, running for 10ms should use 500 gas. For the metrics to be less than 1, the actual `compUsed` should be higher than 500, but in that case, isn't it uses more resource instead of less resource?

Reply:
During the Variable execution effort flip the execution effort weights were set so that a transaction with max computation (9999 c) would run for 200 ms. and that is the EstimatedComputationPerMillisecond (=9999 c/ 200 ms).

If a transaction runs for 10 ms and uses 500 computation that means `NormalizedTimePerComputation = (10 ms / 500 c) * ( 9999 c / 200 ms) = 0.9999` which means its was run as fast as we would expect it to run. A transaction using 500 c but running 20 ms would have a `NormalizedTimePerComputation`  of about 2 which means it took twice as long as expected. But if it ran wit 5 ms it would have a value of 0.5 which would mean it ran twice as fast as expected.

> For the metrics to be less than 1, the actual compUsed should be higher than 500, but in that case, isn't it uses more resource instead of less resource?

the amount of time spent on the transaction is the resource here. So if a transaction ran for 10 ms that is the resources it used. Computation used is an estimate for time spent. We estimate transactions that run 10 ms to be worth about 500 computation. If instead the transaction was worth 1000 computation than we ran it faster then expected (it was estimated to run for 20ms). If the transaction was worth 250 computation than it ran slower than expected (it was estimated to run 5ms).